src/login/login.component.tsx

```diff
@@ -3,8 +3,10 @@ import { performLogin } from "./login.resource";
 import { always } from "kremling";
 import styles from "./login.component.css";
 import { getCurrentUser } from "@openmrs/esm-api";
+import { getConfig } from "@openmrs/esm-config";
```

Comment:
I think there was a miscommunication - I was under the impression we had ultimately decided to put this into esm-root-config. We had several discussions about this and I think i made it confusing by going back and forth on that.

I think that the difference between `esm-config` and `esm-root-config` will be confusing to developers. If we do have it as its own module, I think we'd have to make the name more specific such as `@openmrs/esm-impl-config` so as to help avoid confusion. Maybe let's talk over slack sometime about this? Am interested to hear your thoughts on it.

Reply:
Yeah, I tried that first, see my [comment in Slack](https://openmrs.slack.com/archives/CHP5QAE5R/p1570310563260200) (copy-pasted below)

> Ok, I've produced the first failure on the way to creating the config library. It appears that the config library must be loaded through the import map in order to share state (configurations) across modules. This means that it can't live in openmrs-esm-root-config, which has to be imported as a compile-time dependency by overriding root configs. I'm going to start a new module for the config library unless someone tells me I shouldn't.
To see the point I got to, see
https://github.com/openmrs/openmrs-esm-root-config/tree/MF-52-config-skel
https://github.com/PIH/pih-esm-root-config and
https://github.com/openmrs/openmrs-esm-login/tree/MF-52-config-skel

Reply:
Good point about the naming being confusing. `@openmrs/esm-module-config`? Since it is the thing that should be used to configure modules, while the root-config could be construed as a thing that configures the implementation as a whole?

Reply:
👍 I like `esm-module-config`